public/styles/styles.css

```diff
@@ -1,10 +1,100 @@
 body {
   color: #333;
   font-family: Helvetica, Arial, sans-serif;
-  background-color: skyblue; /* Sanity Check! */
+  /*background-color: skyblue; /* Sanity Check! */
 }
 
 h1 {
-  margin-top: 100px;
+  margin-top: 50px;
   text-align: center;
 }
+/*from portfolio start*/
```

Comment:
you could organize this CSS file by order of specificity -- i.e. styles that apply to the entire page first and then smaller elements

Reply:
yeah. Thanks, I ll work on it.